include/stl2/detail/algorithm/is_permutation.hpp

```diff
@@ -192,8 +192,9 @@ STL2_OPEN_NAMESPACE {
 		}
 		return __stl2::is_permutation(
 			__stl2::begin(rng1), __stl2::end(rng1),
-			__stl2::begin(rng2), std::ref(pred),
-			std::ref(proj1), std::ref(proj2));
+			__stl2::begin(rng2), __stl2::end(rng2),
+			std::ref(pred), std::ref(proj1),
+			std::ref(proj2));
```

Comment:
This change removes the optimization which was the original motivation for having this `SizedRange` overload. It would be better to yank the body out of the deprecated overload on line 72, give it an internal-only name, and have both this overload and line 72 forward into that internal-only backend.